attributes/ckan.rb

```diff
@@ -22,7 +22,7 @@
 #
 
 default['datashades']['ckan_web']['endpoint'] = '/'
-default['datashades']['ckan_web']['packages'] = ['xml-commons', 'git', 'postgresql94-devel', 'postgresql94', 'python27-devel', 'python27-pip', 'libxslt', 'libxslt-devel', 'libxml2', 'libxml2-devel', 'gcc', 'gcc-c++', 'make', 'java-1.8.0-openjdk-devel', 'java-1.8.0-openjdk', 'xalan-j2', 'unzip', 'policycoreutils-python', 'mod24_wsgi-python27', 'supervisor', 'squid']
```

Comment:
was java added to the solr install cookbook or did it already exist? i could not see it in this pr

Reply:
We haven't needed it ever since we moved Solr onto the service layer, years ago. Just a cleanup. (And yes, the Solr recipes include it, see `attributes/solr.rb`.)